### PR TITLE
update dependency for libxdp/libbpf

### PR DESCRIPTION
**Pull request by ami-GS**
## Description

Remove dependency to xdp-tools. install libxdp/libbpf headers instead.

## Testing

N/A

## Documentation

N/A

### CODE REVIEW

File: .docker/ubuntu-20.04/Dockerfile [hunk 1]
```diff
@@ -39,6 +40,17 @@ RUN apt-get update && apt-get install --no-install-recommends -y apt-transport-h
     cppcheck \
     clang-tidy \
     gdb \
+    libxdp-dev \
```
Comment (nibanks) [inline .docker/ubuntu-20.04/Dockerfile:43]: You mentioned in our teams call today that some packages were old versions. Do these have the versions you need?

Comment (ami-GS) [reply .docker/ubuntu-20.04/Dockerfile:43]: `apt-add-repository "deb http://mirrors.kernel.org/ubuntu noble main"`
I found this repository having 1.4.0. works on Ubuntu 22.04

Comment (ami-GS) [reply .docker/ubuntu-20.04/Dockerfile:43]: by default there is no libxdp package at all. there is only libbpf 0.5 or so (ubuntu 22.04)

Comment (ami-GS) [reply .docker/ubuntu-20.04/Dockerfile:43]: xdp-tools seems to be not providing binary.
https://github.com/xdp-project/xdp-tools/issues/416

**Review by nibanks — APPROVED** [review]

**Comment by ami-GS** [comment]
libxdp/libbpf and libcrypt1 conflicts libc6 version.....

**Review by nibanks — APPROVED** [review]

**Pull request merged**